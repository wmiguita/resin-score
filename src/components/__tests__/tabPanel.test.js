import React from "react"
import renderer from "react-test-renderer"

import TabPanel from "../tabPanel"

describe( "TabPanel", () => {
  it( "should render TabPanel", () => {
    const tree = renderer.create( <TabPanel index={ 1 } value={ 2 }><input /></TabPanel> ).toJSON()

    expect( tree ).toMatchSnapshot()
  })

  it( "should be hidden when value and index are different", () => {
    const tab = renderer.create( <TabPanel index={ 0 } value={ 1 } ><input /></TabPanel> )

    const div = tab.root.findByType( 'div' ) //paper material-ui component

    expect( div.props.hidden ).toBe( true )
  })
})
