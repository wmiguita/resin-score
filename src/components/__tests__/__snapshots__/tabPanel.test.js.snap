// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabPanel should render TabPanel 1`] = `
<div
  hidden={false}
  role="tabpanel"
>
  <input />
</div>
`;
